style.css

```diff
@@ -515,3 +519,326 @@ footer {
   bottom: 0;
   right: -10px;
 }
+
+@media (min-width: 375px) {
+  .about-me__paragraph {
+    width: 85%;
+  }
+
```

Comment:
Please, ensure that the mobile version is responsive until the desktop version. This will allow for a good user experience across all screen sizes.

Your design:
![image](https://user-images.githubusercontent.com/20554583/228993147-1c7c3eaa-3de6-4b4f-9fd6-9a3f9e28cfc0.png)